src/main/resources/templates/db_scripts/schema.json

```diff
@@ -0,0 +1,10 @@
+{
+  "tables": [],
+  "scripts": [
+    {
+      "run": "after",
+      "snippetPath": "create_records_orders_table.sql",
+      "fromModuleVersion": "mod-source-record-manager-12.6.0"
```

Comment:
should this reference mod-orders instead?

Reply:
sure, thanks!